### DISCUSSION TITLE
Funding/grants for accessibility

### DISCUSSION THREAD

**Discussion by isabela-pf** [jupyter/accessibility#26] [Ideas]
Based on multiple JupyterLab Accessibility meetings and @saulshanabrook's [briefing](https://github.com/saulshanabrook/jupyterlab-accessibility/issues/38), I'm starting this discussion as a place to store thoughts on
- Potential funding opportunities
- Ideas for funded work
- Progress on funding being pursued
- Anything else related to the topic!

Links to meeting minutes that discuss this topic:
to be added

**Comment by choldgraf** [comment]
I think that JupyterLab Accessibility improvements could be a strong grant for this EOSS cycle. Two roles that I could imagine:

- Fund a person's time (maybe @isabela-pf ?) to help with wrangling the community around a11y
- Fund a developer's time to focus their efforts on a11y improvements

I am happy to help folks that are interested in putting together a grant application for this round (or more generally, helping find funding for this)
